packages/@aws-cdk/aws-sns-subscriptions/lib/subscription.ts

```diff
@@ -11,6 +11,18 @@ export interface SubscriptionProps {
    * @default - all messages are delivered
    */
   readonly filterPolicy?: { [attribute: string]: sns.SubscriptionFilter };
```

Comment:
Adding a `@deprecated` to the jsdoc causes errors in the unit tests as any use of `filterPolicy` throws an error. I believe that is the correct practice but would this not break any existing uses of `filterPolicy`?